Fixtures/IncrementalBuildTests/SourceCodeError/Sources/main.swift

```diff
@@ -0,0 +1,2 @@
+print(firstLine)
+print(secondLine)
```

Comment:
They're similar, but not identical. This test makes a syntax error in a source code file ("SecondLine.swift"), whereas the PackageError test makes a syntax error in "Package.swift".